components/Footer.js

```diff
@@ -1,9 +1,48 @@
 import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
 
-export default function Footer() {
+export default function Footer({
+  activePage,
+  setActivePage,
+  toggleActivePage,
+}) {
+  const router = useRouter();
```

Comment:
If this would be called [isPageActive , setIsPageActive] it would be more clear what this is about.
Reading "activePage" I would expect to see a string which page is the active page.